manifests/05-dedicated-admins-logging.ClusterRole.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Is this too broad? Why is it a `ClusterRole` and not a `Role`? Will the permissions in this role be used in other namespaces?

Reply:
Only used in RoleBinding in openshift-logging NS.